tests/test_events.py

```diff
@@ -0,0 +1,695 @@
+
+import mock
+from nose.tools import *
+from collections import OrderedDict
+
+from website.notifications.events.model import *
```

Comment:
Except for `from nose.tools import *`, "*" imports should be avoided, as they make it difficult to track where symbols come from without namespacing. Import the `model` and `utils` modules instead.